webnode/src/components/consent-overlay/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+
+import ButtonGroup from "./button-group";
+import MiddleGroup from "./middle-group";
+import LogoGroup from "./logo-group";
+import * as WebFont from "webfontloader";
+import Radium from 'radium';
+
+WebFont.load({
+  google: {
+    families: ["Poppins:400,600", "sans-serif"]
+  }
+});
+
+interface ConsentOverlayProps {
+  giveConsent: () => any;
+  denyConsent: () => any;
+  status: string;
+}
+
+interface State {
```

Comment:
i dont think we should create state interfaces if we are not using them, does the TS compiler throw an error without this?

Reply:
we dont need init if we dont use <>, props should be defined - react need it, for states can be <>